File: IntegrationFinale/User/src/entity/Activite.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package entity;

import java.sql.Date;
import java.sql.Time;

/**
 *
 * @author ASUS
 */
public class Activite {
    private int IdActivite;
    private String Intitule;
    private String Niveau;
    private String Responsable;
    private String Type;
    private String HeureDebut;
    private String HeureFin;

    public Activite() {
    }
   public Activite( String Intitule, String Niveau, String Responsable, String Type, String HeureDebut, String HeureFin) {
        this.Intitule = Intitule;
        this.Niveau = Niveau;
        this.Responsable = Responsable;
        this.Type = Type;
        this.HeureDebut = HeureDebut;
        this.HeureFin = HeureFin;
    }
    
    public Activite(int IdActivite, String Intitule, String Niveau, String Responsable, String Type, String HeureDebut, String HeureFin) {
        this.IdActivite = IdActivite;
        this.Intitule = Intitule;
        this.Niveau = Niveau;
        this.Responsable = Responsable;
        this.Type = Type;
        this.HeureDebut = HeureDebut;
        this.HeureFin = HeureFin;
    }

    public String getHeureDebut() {
        return HeureDebut;
    }

    

    

  

    public String getHeureFin() {
        return HeureFin;
    }

    public int getIdActivite() {
        return IdActivite;
    }

    public String getIntitule() {
        return Intitule;
    }

    public String getNiveau() {
        return Niveau;
    }

    public String getResponsable() {
        return Responsable;
    }

    public String getType() {
        return Type;
    }

    public void setHeureDebut(String HeureDebut) {
        this.HeureDebut = HeureDebut;
    }

    public void setHeureFin(String HeureFin) {
        this.HeureFin = HeureFin;
    }

    
  

    public void setIdActivite(int IdActivite) {
        this.IdActivite = IdActivite;
    }

    public void setIntitule(String Intitule) {
        this.Intitule = Intitule;
    }

    public void setNiveau(String Niveau) {
        this.Niveau = Niveau;
    }

    public void setResponsable(String Responsable) {
        this.Responsable = Responsable;
    }

    public void setType(String Type) {
        this.Type = Type;
    }

    @Override
    public String toString() {
        return "Activite{" + "IdActivite=" + IdActivite + ", Intitule=" + Intitule + ", Niveau=" + Niveau + ", Responsable=" + Responsable + ", Type=" + Type + ", HeureDebut=" + HeureDebut + ", HeureFin=" + HeureFin + '}';
    }

  
   


}

File: IntegrationFinale/PiDevF2/PiDev/src/PidevBundle/Entity/Activite.php
<?php

namespace PidevBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Activite
 *
 * @ORM\Table(name="activite")
 * @ORM\Entity
 */
class Activite
{
    /**
     * @var integer
     *
     * @ORM\Column(name="IdActivite", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $idactivite;

    /**
     * @var string
     *
     * @ORM\Column(name="Intitule", type="string", length=30, nullable=false)
     */
    private $intitule;

    /**
     * @var string
     *
     * @ORM\Column(name="Niveau", type="string", length=30, nullable=false)
     */
    private $niveau;

    /**
     * @var string
     *
     * @ORM\Column(name="Responsable", type="string", length=30, nullable=false)
     */
    private $responsable;

    /**
     * @var string
     *
     * @ORM\Column(name="Type", type="string", length=30, nullable=false)
     */
    private $type;

    /**
     * @var string
     *
     * @ORM\Column(name="HeureDebut", type="text", nullable=false)
     */
    private $heuredebut;

    /**
     * @var string
     *
     * @ORM\Column(name="HeureFin", type="string", length=30, nullable=false)
     */
    private $heurefin;

    /**
     * Activite constructor.
     * @param int $idactivite
     * @param string $intitule
     * @param string $niveau
     * @param string $responsable
     * @param string $type
     * @param string $heuredebut
     * @param string $heurefin
     */


    /**
     * @return int
     */
    public function getIdactivite()
    {
        return $this->idactivite;
    }

    /**
     * @param int $idactivite
     */


    /**
     * @return string
     */
    public function getIntitule()
    {
        return $this->intitule;
    }

    /**
     * @param string $intitule
     */
    public function setIntitule($intitule)
    {
        $this->intitule = $intitule;
    }

    /**
     * @return string
     */
    public function getNiveau()
    {
        return $this->niveau;
    }

    /**
     * @param string $niveau
     */
    public function setNiveau($niveau)
    {
        $this->niveau = $niveau;
    }

    /**
     * @return string
     */
    public function getResponsable()
    {
        return $this->responsable;
    }

    /**
     * @param string $responsable
     */
    public function setResponsable($responsable)
    {
        $this->responsable = $responsable;
    }

    /**
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType($type)
    {
        $this->type = $type;
    }

    /**
     * @return string
     */
    public function getHeuredebut()
    {
        return $this->heuredebut;
    }

    /**
     * @param string $heuredebut
     */
    public function setHeuredebut($heuredebut)
    {
        $this->heuredebut = $heuredebut;
    }

    /**
     * @return string
     */
    public function getHeurefin()
    {
        return $this->heurefin;
    }

    /**
     * @param string $heurefin
     */
    public function setHeurefin($heurefin)
    {
        $this->heurefin = $heurefin;
    }
    public function __toString()
    {
        return (string) $this->getHeurefin();
    }


}

